src/app/service/authentication.service.ts

```diff
@@ -11,6 +11,8 @@ import { HttpClient } from './http-client';
 })
 export class AuthenticationService {
   private returnUrl: string;
+  public users$: Observable<SumzAPI[]>;
```

Comment:
Wenn der AuthenticationService Observables einsetzt, sollte er das auch überall tun (und die Observables nicht auf `SumzAPI[]`, sondern auf `User` typisieren). Bin mir nur nicht sicher, ob das in diesem Fall wirklich sooo wichtig ist.